src/sscofpmf.adoc

```diff
@@ -45,6 +45,8 @@ The following bits are added to `mhpmevent`:
 | UINH | If set, then counting of events in U-mode is inhibited
 | VSINH | If set, then counting of events in VS-mode is inhibited
 | VUINH | If set, then counting of events in VU-mode is inhibited
+| _WPRI_ | Reserved
+| _WPRI_ | Reserved
```

Comment:
In the first column, I would put the bit range/position rather than WPRI.  Could just be a single row:
`| 57:56 | Reserved`

Reply:
But per the last ARC meeting it was specifically resolved that these two bist should be Reserved and declared as WPRI (i.e not just Reserved with no further declaration to software and hardware).

Reply:
I'm not disagreeing with that.  The diagram above this lists 57:56 as WPRI, but this table below is giving definitions per bit/field.  So putting 57 and 56, rather than WPRI and WPRI, in the first column makes more sense.  Then it's clear that 57:56 are WPRI and Reserved.

Reply:
That makes sense.  I'm agnostic to treating these two bist as two bits or as one bit field.  For clarity to Bill (as to what he should do), go ahead and make a comment on the PR to merge these two bits into one Reserved and WPRI bit field.

Reply:
In which case, I suggest @bcstrongx provide a clear bottom-line net direction - as a comment to the PR - of what the final version of the Smcntrpmf text should look like.

Reply:
In our side email thread, we agreed that WPRI implies that the associated bits are reserved.  So I would probably remove rows 48 & 49 above, but don't feel strongly.  If it is preferred to more explicitly call out that these bits are reserved, I would replace lines 48 & 49 with `| 57:56 | Reserved`

Reply:
Thanks, I have removed the changes from Smcntrpmf, with the exception of an extra space at the end of a line.